File: src/main/java/com/prography/zone_2_be/Zone2BeApplication.java
package com.prography.zone_2_be;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.security.servlet.UserDetailsServiceAutoConfiguration;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication(exclude = UserDetailsServiceAutoConfiguration.class)
@EnableScheduling
public class Zone2BeApplication {
	public static void main(String[] args) {
		SpringApplication.run(Zone2BeApplication.class, args);
	}
}

File: src/main/kotlin/com/prography/zone_2_be/Zone2BeApplication.kt
package com.prography.zone_2_be

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity

@EnableWebSecurity
@SpringBootApplication
class Zone2BeApplication

fun main(args: Array<String>) {
	runApplication<Zone2BeApplication>(*args)
}
